packages/core/useInfiniteScroll/index.ts

```diff
@@ -74,6 +74,7 @@ export function useInfiniteScroll(
         ])
           .finally(() => {
             promise.value = null
+            state.measure()
```

Comment:
Maybe we could put this at the beginning of `checkAndLoad`? Here I think it's better to do after nextTick